server/graphql/types/src/types/store.rs

```diff
@@ -47,6 +47,10 @@ impl StoreNode {
     pub async fn site_id(&self) -> i32 {
         self.row().site_id
     }
+
+    pub async fn logo(&self) -> &str {
```

Comment:
Could you please add some docs comment here, e.g.:
```suggestion
    /// Returns the associated store logo.
    /// The logo is returned as a data URL schema, e.g. "data:image/png;base64,..."
    pub async fn logo(&self) -> &str {
```

This comment will then automatically show up in the graphql api documentation.

What if there is no logo for a store? in this case the API should return `&Option<String>`...